src/ert/plugins/__init__.py

```diff
@@ -27,6 +27,7 @@ def inner(*args: P.args, **kwargs: P.kwargs) -> Any:
                     "installable_workflow_jobs",
                     "help_links",
                     "installable_forward_model_steps",
+                    "forward_model_configuration",
                     "ecl100_config_path",
                     "ecl300_config_path",
                     "flow_config_path",
```

Comment:
I guess the goal should be to get rid of these hooks as well? :thinking:

Reply:
Yes, but that will be in a separate PR.